src/layout/layout.js

```diff
@@ -303,7 +303,7 @@
 
           var drawerButtonIcon = document.createElement('i');
           drawerButtonIcon.classList.add(this.CssClasses_.ICON);
-          drawerButtonIcon.textContent = this.Constant_.MENU_ICON;
+          drawerButtonIcon.innerHTML = this.Constant_.MENU_ICON;
```

Comment:
I’m security n00b. Could you explain to me how anything can get evaluated if I block replace `<` and `>` with `&lt;` and `&gt;` respectively?